lxd/main_init.go

```diff
@@ -124,6 +124,14 @@ func (c *cmdInit) Run(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	// If clustering is enabled, and no cluster.https_address network address
+	// was specified, we fallback to core.https_address.
```

Comment:
Why is this done in the client rather than on the server side?
If the client needs this logic, then it's breaking backward compatibility with older clients which would be unfortunate.

Reply:
In this case the client just uses the new API, the old API still works. If you prefer, I can have main_init.go use the old API.